packages/bootstrap/scss/_variables.scss

```
@@ -1556,7 +1556,7 @@ $drawer-item-padding-x: $padding-x !default;
 $drawer-item-padding-y: $padding-y !default;
 $drawer-item-font-size: 16px !default;
 $drawer-item-line-height: $line-height-lg !default;
-$drawer-icon-size: 18px !default;
+$drawer-icon-size: 16px !default;
```

There is already an `$icon-size` variable. Use it.
```suggestion
$drawer-icon-size: $icon-size !default;
```